samples/sensor/grow_r502a/boards/frdm_k64f.overlay

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2021, Linumiz
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&uart3 {
+        current-speed = <57600>;
+        status = "okay";
+        grow_r502a {
+                compatible = "hzgrow,r502a";
+                int-gpios = <&gpioc 12 GPIO_ACTIVE_LOW>;
+        };
+};
```

Comment:
please use tabs, no spaces.

Reply:
removed this overlay, as one overlay is enough for a sample.